src/configuration/routes/cache.md

```diff
@@ -1,16 +1,35 @@
-# HTTP Cache
+# HTTP cache
 
-## Rationale
+Platform.sh supports HTTP caching at the server level. The cache is only applied to `GET` and `HEAD` requests.
 
-Many hosting solutions are adding an additional layer (Varnish...) to handle caching. Platform.sh supports HTTP caching right at the web server level.
+The cache must be explicitly enabled using the `cache` key in your `.platform/routes.yaml` file. Platform.sh serves the files or requests generated by the application directly, unless the cache is enabled.
```

Comment:
Actually the cache is enabled by default (if the `cache` key does not exist for the route), with the following settings:

```yaml
cache:
    enabled: true
    default_ttl: 0
    cookies: ['*']
    headers: ['Accept', 'Accept-Language']
```